components/tyk-dev-portal/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0-beta4
+version: 1.0.0-beta1
```

Comment:
This is not a request for a change but maybe we should raise a ticket to consider removing old `tyk-enterprise-portal` chart from helm repository (cloudsmith).

Reply:
Yes, this can be done whenever we want post release but we need help from someone with access.